test/demo/index.html

```diff
@@ -303,6 +303,11 @@ <h3>Timeseries (secondly)</h3>
           <div class="chart-c3-area-spline-time-stacked keen-chart-container"></div>
         </div>
       </div>
+      <div class="keen-ui-row">
+        <div class="keen-ui-col-4">
+          <div class="chart-c3-area-spline-metrics keen-chart-container"></div>
```

Comment:
let's use custom width & height - the same as in the auto-collector-dashboard repo:
width: 270px
height: 120px